### PR TITLE
done some translation

### PR DESCRIPTION
**Pull request by sunshine1126**
Hello, I have finished my translation project and please check my assignment for translation, thanks.

### CODE REVIEW

**Comment by MichaelChirico** [comment]
Looks good from my end, thanks! @soappp9527 could you help by reviewing the translations?

**Comment by MichaelChirico** [comment]
@sunshine1126 please address the revisions requested by @soappp9527 when you get a chance, thanks! :)

File: po/R-zh_CN.po [hunk 1]
```diff
@@ -1730,72 +1730,88 @@ msgid "using"
...
+"这个可以通过查看软件包源文件中的po/R-<locale>.po和po/<locale>.po文件获得，"
+"这个文件可以并排找到母语和英语错误信息。"
 
 msgid ""
 "**********\n"
 "This development version of data.table was built more than 4 weeks ago. "
 "Please update: data.table::update.dev.pkg()\n"
 "**********"
-msgstr ""
+msgstr "**********"
```
[the remark this answers is not in the document]
Comment (MichaelChirico) [reply po/R-zh_CN.po]: yes, you should include the `\n`. can just include exactly as in the original message.

File: po/zh_CN.po [hunk 2]
```diff
@@ -3497,222 +3497,222 @@ msgstr ""
...
 "Internal error: invalid value for 'type'; this should have been caught "
 "before. please report to data.table issue tracker"
-msgstr ""
+msgstr "内部错误：'type'的值无效；这应该在之前被找到。请报告给data.table问题跟踪器"
 
 #: ijoin.c:56 ijoin.c:121
 #, c-format
 msgid "Internal error: unknown type in mult=%d in lookup: %d"
-msgstr ""
+msgstr "内部错误：在查找mult=%d的过程中出现未知的类型：%d"
```
[the remark this answers is not in the document]
Comment (MichaelChirico) [reply po/zh_CN.po]: Hi @sunshine1126 you may have missed this one

File: po/zh_CN.po [hunk 3]
```diff
@@ -3497,222 +3497,222 @@ msgstr ""
...
 "Internal error: unknown type in mult=%d in lookup should have been caught "
 "earlier: %d"
-msgstr ""
+msgstr "内部错误：在查找mult=%d的过程中出现未知的类型应该被更早找到：%d"
 
 #: ijoin.c:220
 #, c-format
 msgid "Final step in generating lookup ... done in %8.3f seconds\n"
-msgstr ""
+msgstr "生成查找的最后一步...在在%8.3f秒内完成"
```
[the remark this answers is not in the document]
Comment (MichaelChirico) [reply po/zh_CN.po]: @sunshine1126 also this

File: po/R-zh_CN.po [hunk 4]
```diff
@@ -1807,57 +1823,69 @@ msgid ""
...
+"R和C代码之间的这种不匹配可能发生在任何包中，而不仅仅是在data.table中。"
+"只是data.table添加了这个检查"
 
 msgid "This is R"
-msgstr ""
+msgstr "这是R"
 
 msgid "but data.table has been installed using R"
-msgstr ""
+msgstr "但是data.table已经使用R被安装"
```
[the remark this answers is not in the document]
Comment (soappp9527) [reply po/R-zh_CN.po]: you should combine this with the last sentence and padding with R version number. for example, `This is R 3.6.1 but data.table has been installed using R 3.5.2`. 
So if chinese translation is like `这是R 3.6.1 但是data.table已经使用R被安装 3.5.2`, that will be really confusing, right?

[the remark this answers is not in the document]
Comment (soappp9527) [reply po/R-zh_CN.po]: you should combine this with the last sentence and padding with R version number. for example, `This is R 3.6.1 but data.table has been installed using R 3.5.2`. 
So if chinese translation is like `这是R 3.6.1 但是data.table已经使用R被安装 3.5.2`, that will be really confusing, right?

**Comment by MichaelChirico** [comment]
Am going to go ahead and merge. Thanks @sunshine1126 and @soappp9527 !!!

File: po/zh_CN.po [hunk 5]
```diff
@@ -3617,213 +3617,216 @@ msgstr ""
...
 #, c-format
 msgid "Internal error: unknown mult in overlaps: %d"
-msgstr ""
+msgstr "内部错误：在重叠中出现未知的mult：%d"
 
 #: ijoin.c:727
 #, c-format
 msgid "Final step, fetching indices in overlaps ... done in %8.3f seconds\n"
-msgstr ""
+msgstr "重叠的最后一步：获取索引...在％8.3f秒内完成"
```
Comment (MichaelChirico) [inline po/zh_CN.po]: wow, subtle issue uncovered here! That's the wrong `%` 😛

**Pull request merged**